docs/core/porting/winforms.md

```diff
@@ -31,7 +31,9 @@ In this article, various names are used to identify types of files used for migr
 - [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) 3.0 or later.
```

Comment:
a few more comments:
Given that now we're adding the requirement to use Preview 1 here, then pre-reqs should state the version and link to the preview download. Also, since VS and workloads are a pre-req, you shouldn't need to install the SDK separately.